packages/svelte/tests/runtime-legacy/samples/binding-backflow/_config.js

```diff
@@ -34,14 +34,14 @@ export default test({
 
 		p = parents['reactive_mutate'];
 		assert.deepEqual(p.value, { foo: 'kid' });
-		assert.equal(p.updates.length, 1);
+		assert.equal(p.updates.length, 2);
 
 		p = parents['init_update'];
 		assert.deepEqual(p.value, { foo: 'kid' });
 		assert.equal(p.updates.length, 2);
 
 		p = parents['init_mutate'];
 		assert.deepEqual(p.value, { foo: 'kid' });
-		assert.equal(p.updates.length, 1);
+		assert.equal(p.updates.length, 2);
```

Comment:
This test is from #8992 and was (turns out, wrongfully) changed when moving to proxies in #9826 . This uncovered that using `$.proxy` inside `svelte/legacy` could also have caused these kinds of problems this PR fixes in some situations, too, but it shouldn't matter anymore now that deeply reactive state is properly handled in the places where it matters.